lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -86,11 +86,11 @@ def self.init_vmpooler()
         execute_rake_task("beaker_quickstart:gen_hosts[vmpooler]")
       end
 
-      # Print a message to the console and exit with 0
+      # Print a message to the console and exit with 1
       # @param [String] msg the message to print
```

Comment:
If this method is intended for failures, then it could have an intention-revealing name and be something like `fail`, `fail_with`, `fail_beaker`, `fail_subcommand`, etc.  Having the exit code default to `1` would make sense in this case.

If this method is intended as a general way for subcommands to exit, then maybe something like `exit`, `exit_with`, `exit_beaker`, etc.  Since this doesn't assume failure, we'd probably want the exit code to default to `0`.

Either way I think both the message and exit code should be parameters.